docs/configuration/index.md

```diff
@@ -898,7 +898,8 @@ A sample Coordinator dynamic config JSON object is shown below:
   "decommissioningMaxPercentOfMaxSegmentsToMove": 70,
   "pauseCoordination": false,
   "replicateAfterLoadTimeout": false,
-  "maxNonPrimaryReplicantsToLoad": 2147483647
+  "maxNonPrimaryReplicantsToLoad": 2147483647,
+  "maxSegmentsToLoad": 2147483647
```

Comment:
Should the name be more specific? "Max segments to load", by itself, sounds like "the maximum number of segments which the coordinator will load ... period" -- an upper limit on the number of loaded segments overall. This then raises the question, "but, what happens to the others?"

From the description, it sounds like this is "per Coordination run". So, should the name be something like `maxSegmentsPerCoordination`?

Reply:
+1 for a more descriptive name, thanks for calling it out. Will have a change there in a forthcoming commit.